esp-link/main.c

```diff
@@ -79,7 +79,19 @@ static void ICACHE_FLASH_ATTR prHeapTimerCb(void *arg) {
 }
 #endif
 
+# define VERS_STR_STR(V) #V
+# define VERS_STR(V) VERS_STR_STR(V)
+char* esp_link_version = VERS_STR(VERSION);
+
 void user_rf_pre_init(void) {
+  system_set_os_print(DEBUG_SDK);
+#if defined(STA_SSID) && defined(STA_PASS)
+  struct station_config stconf;
+  os_strncpy((char*)stconf.ssid, VERS_STR(STA_SSID), 32);
+  os_strncpy((char*)stconf.password, VERS_STR(STA_PASS), 64);
+  stconf.bssid_set = 0;
+  wifi_station_set_config_current(&stconf);
+#endif
```

Comment:
This creates an inconsistency between what's in the config flash and what the radio is configured with. What does the Wifi web page and the home page show? Also, making a change on the Wifi web page essentially has no effect, which is rather confusing.